packages/app/source/server/projections/open-appeals-projection.js

```diff
@@ -0,0 +1,36 @@
+Space.eventSourcing.Projection.extend(Donations, 'OpenAppealsProjection', {
```

Comment:
ah nice, it's quote time! "premature optimization …" :stuck_out_tongue:

Jokes beside, there is a really bad impact: two extra files + mappings for (atm) no good reason. I would argue the other way: we can split that up into multiple projections at any time later on.

Atm im looking for patterns to reduce the need for these boilerplate-like stuff. Those are IMO the number 1 enemies of Space / ES architecture as everything feels really tedious without any good reason.